components/concurrency_manager/Cargo.toml

```diff
@@ -20,8 +20,8 @@ txn_types = { path = "../txn_types", default-features = false }
 fail = "0.4"
 
 [dependencies.crossbeam-skiplist]
-git = "https://github.com/crossbeam-rs/crossbeam.git"
-tag = "crossbeam-0.7.3"
+git = "https://github.com/sticnarf/crossbeam.git"
+branch = "tikv"
```

Comment:
I think we should use a ref rather than a branch to ensure we're all building the same version